RoadRunner/src/main/java/com/acmerobotics/roadrunner/ftc/OTOS.kt

```diff
@@ -0,0 +1,77 @@
+package com.acmerobotics.roadrunner.ftc
+
+import com.qualcomm.hardware.sparkfun.SparkFunOTOS
+import com.qualcomm.hardware.sparkfun.SparkFunOTOS.Pose2D
+import com.qualcomm.robotcore.hardware.DcMotorSimple
+import com.qualcomm.robotcore.hardware.HardwareDevice
+import com.qualcomm.robotcore.hardware.IMU
+import org.firstinspires.ftc.robotcore.external.navigation.AngleUnit
+import org.firstinspires.ftc.robotcore.external.navigation.AngularVelocity
+import org.firstinspires.ftc.robotcore.external.navigation.AxesOrder
+import org.firstinspires.ftc.robotcore.external.navigation.AxesReference
+import org.firstinspires.ftc.robotcore.external.navigation.Orientation
+import org.firstinspires.ftc.robotcore.external.navigation.Quaternion
+import org.firstinspires.ftc.robotcore.external.navigation.YawPitchRollAngles
+
+enum class EncoderDirection {
+    PARALLEL,
+    PERPENDICULAR
+}
+
+class OTOSView(val otos: SparkFunOTOS) {
+    var pose = Pose2D()
+        private set
+
+    var vel = Pose2D()
+        private set
+
+    fun update() = otos.getPosVelAcc(pose, vel, Pose2D())
+}
+
+class OTOSEncoder(val view: OTOSView, val ed: EncoderDirection) : Encoder {
```

Comment:
I'd rather have separate parallel/perpendicular classes to match the pinpoint implementation and avoid the need for an `EncoderDirection` enum in this file.